pom.xml

```diff
@@ -111,6 +112,11 @@
     </dependency>
     <dependency>
       <groupId>org.pac4j</groupId>
+      <artifactId>j2e-pac4j</artifactId>
+      <version>${j2e-pac4j.version}</version>
```

Comment:
I asked myself the same question, but it would meat to be sure none of the classes references both of the dependencies at the same time, and yes, it is tricky.
Also, I think it's ok, someone will most certainly need the `jax-rs-pac4j` one, and the `j2e-pac4j` one won't introduce much more dependencies (since the servlet api is already here because of dropwizard).